client/extensions/zoninator/components/navigation/index.jsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { find, flowRight, get, trimEnd } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import SectionNav from 'components/section-nav';
+import SectionNavTabs from 'components/section-nav/tabs';
+import SectionNavTabItem from 'components/section-nav/item';
+import SectionNavSegmented from 'components/section-nav/segmented';
+import Button from 'components/button';
+import sectionsModule from 'sections';
+import { getSiteSlug } from 'state/sites/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+const getZones = () => {
```

Comment:
It's a placeholder for now and will eventually be replaced by a Redux selector. I used a function to remind of the dynamic nature of this list.